src/EcommerceStatsBundle/Bridge/Chameleon/Migration/Script/update-1616421500.inc.php

```diff
@@ -0,0 +1,18 @@
+<h1>Build #1616421500</h1>
+<h2>Date: 2020-09-25</h2>
+<div class="changelog">
+    - ref #636: Add routing definitions
+</div>
+<?php
+$data = TCMSLogChange::createMigrationQueryData('pkg_cms_routing', 'de')
+    ->setFields([
+        'name' => 'Ecommerce Stats',
+        'short_description' => 'Export Endpunkte für Statistiken (backend)',
```

Comment:
English translation? Is that field translatable?

Reply:
There are no translatable fields in `pkg_cms_routing`